packages/app/src/cli/services/dev/select-app.ts

```diff
@@ -25,7 +25,7 @@ export async function selectOrCreateApp(
 ): Promise<OrganizationApp> {
   let createNewApp = apps.nodes.length === 0
   if (!createNewApp) {
-    outputInfo(`\nBefore you preview your work, it needs to be associated with an app.\n`)
+    outputInfo(`\nBefore proceeding, your project needs to be associated with an app.\n`)
```

Comment:
We are currently showing this message about `preview your work` not only for the `dev` command, but also for `generate extension` or `function schema`. And we need to use the same for this command as well.

So instead of duplicating the code, I just rephrased the message to make it more generic. How does it sound @MeredithCastile?